Assets/Scripts/UI/Achievements/UIAchievementsMenu.cs

```diff
@@ -79,17 +77,16 @@ private void Awake()
             achievementListItems = new List<UIAchievementButton>();
 
             HideMenu();
-            achievementManager = EOSManager.Instance.GetOrCreateManager<EOSAchievementManager>();
```

Comment:
Explain why _not_ calling this method to get the manager is not a problem - because it does some hookup work for us, and it's concerning that is now circumvented by the singleton pattern.

Reply:
Not calling this method is acceptable, because (now) the base implementation within `ServiceManager` is to add itself as a connect listener, and on dispose remove itself.